test_data/yast/encryption/full_lvm_enc_luks2_no_lvm.yaml

```diff
@@ -0,0 +1,23 @@
+disks:
```

Comment:
could you please update the name, cannot be full lvm and no lvm at the same time.

Reply:
Updated and running a VR https://openqa.opensuse.org/tests/4223620#